package.json

```
@@ -6,9 +6,11 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "preview": "vite preview",
-    "installer": "typeinstall"
+    "installer": "typeinstall",
+    "configure": "sh configure.sh"
   },
   "dependencies": {
+    "inquirer": "^8.2.0",
```

remove this unused package
```suggestion
    "inquirer": "^8.2.0",
    "remove this package"
```